jetstream/tests.py

```diff
@@ -29,20 +30,33 @@ class TestJetstream(TestCase):
     def setUp(self):
         pass
 
+    def _check_cassette(self, cassette, expected_cassette_length):
+        """Make sure the cassette is:
+         1. The expected length and either
+         2. New (dirty) or
+         3. Existing, rewound
+         """
+        self.assertEqual(len(cassette), expected_cassette_length)
+        self.assertTrue(cassette.dirty or cassette.all_played)
```

Comment:
I couldn't immediately figure out how to use it as a decorator. Open to suggestions.